recorded_future/komand_recorded_future/actions/list_url_risk_rules/action.py

```diff
@@ -0,0 +1,27 @@
+import komand
+import requests
+from .schema import ListUrlRiskRulesInput, ListUrlRiskRulesOutput, Output, Component
+# Custom imports below
+
+
+class ListUrlRiskRules(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='list_url_risk_rules',
+                description=Component.DESCRIPTION,
+                input=ListUrlRiskRulesInput(),
+                output=ListUrlRiskRulesOutput())
+
+    def run(self, params={}):
+        try:
+            query_headers = {"X-RFToken": self.connection.token}
+            results = requests.get(
+                "https://api.recordedfuture.com/v2/url/riskrules",
+                headers=query_headers
+            ).json()
+            return {
+                Output.RISK_RULES: results["data"]["results"]
+            }
+        except Exception as e:
```

Comment:
Can we make this a PluginException?

Reply:
Here should be except Exception, because this kind of error is raise by api.
We will add raise PluginException after self.logger.error()